source/connection.cc

```diff
@@ -118,6 +118,19 @@ RestClient::Connection::SetUserAgent(const std::string& userAgent) {
 }
 
 /**
+ * @brief set custom Certificate Authority (CA) path
+ * default restclient-cpp/RESTCLIENT_VERSION string
+ *
+ * @param caInfo - The path to a file holding the certificates used to
+ * verify the peer with. See CURLOPT_CAINFO
+ *
+ */
+void
+RestClient::Connection::SetCAInfo(const std::string& caInfo) {
+  this->caInfo = caInfo;
+}
```

Comment:
Yup, thanks for thinking of this! I thought about it as well and I'm ok with diverging slightly from the libcurl naming to get better clarity into restclient-cpp.